recipes-bsp/acpica/acpica_20190215.bb

```diff
@@ -17,9 +17,9 @@ COMPATIBLE_HOST = "(i.86|x86_64|arm|aarch64).*-linux"
 DEPENDS = "bison flex"
 
 SRC_URI = "https://acpica.org/sites/acpica/files/acpica-unix2-${PV}.tar.gz \
-    "
-SRC_URI[md5sum] = "5aa086f71f4b5273c0932a1e04419a37"
-SRC_URI[sha256sum] = "d1e26cdde58938653a277a5ca59ce1df045508d345fee13fed2eaacc6ef51851"
+           "
```

Comment:
Unrelated change.

Reply:
The change in indent is accidental. The change in checksum due to upgrade to newer acpica.